package.json

```diff
@@ -10,7 +10,7 @@
     "start:dev": "DEBUG='jobadvisor' nodemon ./bin/www -e html,css,dust,js,json,sh",
     "test": "sh test_local.sh",
     "test-fast": "sh test_local_no_download.sh",
-    "test-mocha": "JUNIT_REPORT_PATH=./jenkins-test-results.xml istanbul cover --report cobertura _mocha -- --timeout 3000 -R spec ./test/* --recursive --colors --reporter mocha-jenkins-reporter",
+    "test-mocha": "JUNIT_REPORT_PATH=./jenkins-test-results.xml istanbul cover --report cobertura _mocha -- --timeout 10000 -R spec ./test/* --recursive --colors --reporter mocha-jenkins-reporter",
```

Comment:
Is it really necessary to have 10 seconds? that will take a lot of time for some tests